scripts/zones/Boneyard_Gully/mobs/Swift_Hunter.lua

```diff
@@ -0,0 +1,51 @@
+-----------------------------------
+-- Area: Boneyard Gully
+-- NPC:  Swift_Hunter
+-- ID:     16810002
+-----------------------------------
+-----------------------------------
+
+require("scripts/globals/titles");
+require("scripts/globals/status");
+require("scripts/globals/magic");
+
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:setTP(100);
+    mob:setMobMod(MOBMOD_SUPERLINK,0);
+    mob:speed(0);
+end;
+
+-----------------------------------
+-- onMobEngaged
+-----------------------------------
+function onMobEngaged(mob, target)
+    mob:useMobAbility(22); -- Pit Ambush
+end;
+
+-----------------------------------
+-- onMobWeaponSkill
+-----------------------------------
+
+function onMobWeaponSkill(mob, target)
+    mob:speed(40);

```

Comment:
The antlions are in the ground, hidden. This was meant to prevent them from coming out of the ground until they ambush. I guess it's just a bandaid, but I don't know how to implement a disengage without the mob healing to full.